content/php/concepts/math-functions/terms/round/round.md

```diff
@@ -0,0 +1,60 @@
+# Title:     _round function in php_
+
+## Description
+    The round() function rounds a floating-point number.
+    
+## Tips:
+    | To round a number UP to the nearest integer, look at the ceil() function.
+    | To round a number DOWN to the nearest integer, look at the floor() function.
```

Comment:
[Proofreading] I highly recommend using bold to emphasize specific points. Putting words in all caps comes across as screaming, which can be off-putting to people who are reading your entry. 
```suggestion
    | To round a number **up** to the nearest integer, look at the ceil() function.
    | To round a number **down** to the nearest integer, look at the floor() function.
```